lib/RoadizCoreBundle/src/Repository/NodeTypeRepository.php

```diff
@@ -32,4 +33,16 @@ public function findAll(): array
 
         return $qb->getQuery()->getResult();
     }
+
+    /**
+     * @throws NonUniqueResultException
+     */
+    public function findOneByName(string $name): ?NodeType
+    {
+        return $this->createQueryBuilder('nt')
+            ->where('nt.name = :name')
+            ->setParameter('name', $name)
```

Comment:
Your query is missing `setMaxResults(1)` to enforce that you will have only 1 result